File: Data Structures & Algorithms/reverse-a-linked-list/submission-1.js
/**
 * Definition for singly-linked list.
 * class ListNode {
 *     constructor(val = 0, next = null) {
 *         this.val = val;
 *         this.next = next;
 *     }
 * }
 */

class Solution {
    /**
     * @param {ListNode} head
     * @return {ListNode}
     */
    reverseList(head) {
        let curr = head
        let prev = null
        let next = null
        while (curr) {
            next = curr.next;
            // Reverse pointer
            curr.next = prev;
            // Move prev and curr forward
            prev = curr;
            // console.log(prev.value)
            curr = next;
        }
        return prev
    }
}

File: Data Structures & Algorithms/is-anagram/submission-1.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        left = list(s)
        curr = 0
        print(left)
        while curr <= len(t)-1:
            if t[curr] in left:
                left.remove(t[curr])
                curr+=1
            else:
                return False
        return True

        